src/Scaffolding/VS.Web.CG.Mvc/Templates/MinimalApi/MinimalApiEfNoClass.cshtml

```diff
@@ -47,17 +47,23 @@ public static void @Model.MethodName (this IEndpointRouteBuilder routes)
         {
             return await db.@modelToList;
         })
-@{
-    if(Model.OpenAPI)
-    {
+    @{
+        if(Model.OpenAPI && Model.UseTypedResults)
+        {
         @:.WithName("@getAllModels")
         @:.WithOpenApi();
-    }
-    else
-    {
+        }
+        else if (Model.OpenAPI && !Model.UseTypedResults)
+        {
+        @:.WithName("@getAllModels")
+        @:.WithOpenApi()
+        @:.Produces<@modelList>(StatusCodes.Status200OK);
+        }
+        else
+        {
         @:.WithName("@getAllModels");
+        }
     }
-}
 
         group.MapGet("/{id}", async @typedTaskWithNotFound (@primaryKeyShortTypeName @primaryKeyNameLowerCase, @dbContextName db) =>

```

Comment:
Does this need to be conditioned on `Model.UseTypedResults` too, to ensure that when `TypedResults` is **not** being used, the method has no declared return type or if `@typedTaskWithNotFound` is an empty string, no extra whitespace?

Reply:
I have more changes coming, i'll ping you when its ready for review.